hypershift-operator/metrics.go

```diff
@@ -156,11 +210,24 @@ func (m *hypershiftMetrics) collect(ctx context.Context) error {
 	if err := m.observeNodePools(ctx, &nodePools); err != nil {
 		return err
 	}
+
 	return nil
 }
 
 func setupMetrics(mgr manager.Manager) error {
-	metrics := newMetrics(mgr.GetClient(), mgr.GetLogger().WithName("metrics"))
+	var hypershiftImage ImageInfo
+
+	// We need to create a new client because the manager one still does not have the cache started
+	tmpClient, err := crclient.New(mgr.GetConfig(), crclient.Options{Scheme: hyperapi.Scheme})
+	if err != nil {
+		fmt.Printf("error creating a temporary client: %v\n", err)
```

Comment:
why are you ignoring the error here? this should `return fmt.Errorf`

Reply:
yep, my bad